Assignment 3.Rmd

```diff
@@ -136,18 +142,50 @@ What patterns do you see in the plot?
 It would be useful to determine how many people are in each cluster. We can do this easily with dplyr.
 
 ```{r}
-K7 <- 
+K7 <- count(K5,cluster,week)
+
 ```
 
 Look at the number of people in each cluster, now repeat this process for 3 rather than 2 clusters. Which cluster grouping do you think is more informative? Write your answer below:
 
+Grouping into 3 clusters is in formative. First, clusters is more comparative, because there are around 8 people in each cluster. Also, the former cluster-1 is broke into two sub-cluster, whose trend is totally different in the new picture.
+
 Once you have done this, save both of your plots to the Assignment 5 file. Create a Zotero item to index your new computer program (Assignment 5.rmd) in Zotero. Then commit your assignment, push it to your Github account and then Pull request your version to the original assignment version so I can see it.
 
 ##Part II
 
 Now, try to do the same for the data collected in class. Create two sets of clusters, the answers to the questions and regions where people grew up. 
+```{r}
+library(ggmap)
+
+D1<- read.csv("HUDK405018-cluster.csv", header = TRUE, sep=",")
+
+D2<-D1[-c(1,2),]
+
+D3<- D2[c(18,20:25)]
+
+D4<-filter(D3, Q2_1!=""&Q2_2!=""&Q2_3!=""&Q2_4!=""&Q2_5!=""&Q2_6!="")
+  
+D5<- geocode(as.character(D4$Q1_1), output = "latlon", source = "dsk")
+
+fit2 <- kmeans(D5, 4)
+
+D6<-data.frame(D5, fit2$cluster)
+
+names(D6) <- c("lon", "lat", "cluster")
+
+D7<- D4[2:7]
+
+fit3 <- kmeans(D7, 4)
+```
 
 ##Part III
 
 Create a visualization that shows the overlap between the two groups of clusters you created in part III.
 
+```{r}
+
+ggplot(D6, aes(x=lon, y=lat,shape=as.factor(fit2$cluster), color=as.factor(fit3$cluster) )) +geom_point( size = 2)
+
+```
```

Comment:
Do you observe much overlap between the clusters? Are there groups where geography and survey clusters overlap?